src/python/pants/core/util_rules/environments.py

```diff
@@ -204,12 +204,37 @@ class RemoteExtraPlatformPropertiesField(StringSequenceField):
     )
 
 
+_NO_FALLBACK_ENVIRONMENT = "__none__"
```

Comment:
I also considered `<none>`, which we've used elsewhere. But I figured be consistent with `__local__`?

Reply:
Originally it was `None`, but this causes the default of `__local__` to be used :/

Reply:
to me, `<none>` is clearer that it indicates "no value" than `__none__`, it doesn't have to be consistent with the format of a valid value (almost by definition).